runtime/doc/eval.txt

```diff
@@ -7911,6 +7910,11 @@ shiftwidth()						*shiftwidth()*
 			endif
 <		And then use s:sw() instead of &sw.
 
+		When there is one argument {col} this is used as column number
+		for which to return the 'shiftwidth' value. This matters for the
+		'vartabstop'. If no {col} argument is given, column 1 will be
```

Comment:
Vim has `This matters for the 'vartabstop' feature.`

Keep `feature` or remove `the`.